### ISSUE TITLE
The validation loss in result.csv is always '0'

### ISSUE THREAD

**Issue by Youzi-1999** [WongKinYiu/yolov9#171]
no matter which size of model i choose, the train results file is always '0' at 'val/box_loss', 'val/cls_loss' and 'val/dfl loss'.
I used my own dataset, and this dataset has trained on yolov5 and yolov8, so it might be right. 
And I used the provided command 'python train_dual.py --workers 4 --device 1 --batch 8 --data data/mydata.yaml --img 520 --cfg models/detect/yolov9-c.yaml --name yolov9_c_exp --hyp data/mydata/hyp.scratch-high.yaml --min-items 0 --epochs 300 --close-mosaic 10'.
Did I miss something?

**Comment by Youho99** [comment]
#132

This is not yet implemented... (for yolov9 models, not for gelan)

**Comment by ThanhPham1987** [comment]
I have the same problem.

**Comment by Youzi-1999** [comment]
> #132
> 
> This is not yet implemented... (for yolov9 models, not for gelan)

Thank you! This is what I'm looking for.

**Comment by Youzi-1999** [comment]
> I have the same problem. ![image](https://private-user-images.githubusercontent.com/22832922/310474840-3e6a42fc-14a9-41a2-b79c-8c24fbf2fe16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3Mjg2NDEsIm5iZiI6MTcwOTcyODM0MSwicGF0aCI6Ii8yMjgzMjkyMi8zMTA0NzQ4NDAtM2U2YTQyZmMtMTRhOS00MWEyLWI3OWMtOGMyNGZiZjJmZTE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA2VDEyMzIyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyMzc1Njg0MjA4OTczOGYyNjBiM2JmOGM2ZGY5ZTY4NDM2NWJiNDAzODg1NjU4N2E0ZDhhNTc5YmRmMmUyOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ciziBNw64qxSX84i2p3r5bduy2vidoVq2YhgwnJDFss)

Maybe your problem and mine are different.

**Comment by Youzi-1999** [comment]
To add, I tried to open the loss function calculation of the validation set myself, but an error was reported because the image size did not match.

**Comment by Youzi-1999** [comment]
> I have the same problem. ![image](https://private-user-images.githubusercontent.com/22832922/310474840-3e6a42fc-14a9-41a2-b79c-8c24fbf2fe16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3Mjg2NDEsIm5iZiI6MTcwOTcyODM0MSwicGF0aCI6Ii8yMjgzMjkyMi8zMTA0NzQ4NDAtM2U2YTQyZmMtMTRhOS00MWEyLWI3OWMtOGMyNGZiZjJmZTE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA2VDEyMzIyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyMzc1Njg0MjA4OTczOGYyNjBiM2JmOGM2ZGY5ZTY4NDM2NWJiNDAzODg1NjU4N2E0ZDhhNTc5YmRmMmUyOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ciziBNw64qxSX84i2p3r5bduy2vidoVq2YhgwnJDFss)

the zero in the picture is the train result.

**Comment by roman-forever** [comment]
Hello, I tried to open the loss function calculation of the validation set and also an error was reported because the image size did not match. How did you solve this problem? I need it so much (cry)

**Comment by Youho99** [comment]
What is your command prompt ?

**Comment by roman-forever** [comment]
oh ,I received your reply in the other issue and I will communicate with you there,thank you very very very much
